Userland/Applications/FileManager/main.cpp

```diff
@@ -806,6 +806,18 @@ ErrorOr<int> run_in_windowed_mode(DeprecatedString const& initial_location, Depr
         window);
     copy_action->set_enabled(false);
 
+    auto copy_path_action = GUI::Action::create(
+        "Copy Path", [&](GUI::Action const&) {
+            auto paths = directory_view->selected_file_paths();
+            if (paths.is_empty()) {
+                paths = tree_view_selected_file_paths();
+            }
+            VERIFY(!paths.is_empty());
+
+            GUI::Clipboard::the().set_plain_text(paths[0]);
```

Comment:
Why not support any amount of paths? You could join them with a newline, for example. It feels weird to be able to select multiple files, only to get a single path put in your clipboard.

Bonus points if the context menu item changes to "Copy Paths" when applicable :^)

Reply:
Yes it sounds amazing idea. I have implemented both of them.